helm/azure-operator/templates/service.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: {{ include "resource.default.namespace"  . }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
+    giantswarm.io/monitoring: "true"
```

Comment:
What do you think about adding it as [a _common_ label here](https://github.com/giantswarm/azure-operator/blob/bb6a7330bbe7bf28bd799218b658252d632468f5/helm/azure-operator/templates/_helpers.tpl#L19)?

Reply:
If we do, the label will be everywhere when we only need it on the service so we can scrape it using the new prometheus setup

Reply:
Oh you are right. Never mind me then :smiley_cat:

Reply:
Well, now you know :dog: 
If we actually add it everywhere, we might scrape the endpoints multiple times